### ISSUE TITLE
422: Cannot read property 'forEach' of undefined

### ISSUE THREAD

**Issue by ghalevy** [F5Networks/f5-appsvcs-extension#418]
### Environment
 * Application Services Version: 3.25.0
 * BIG-IP Version: 15.1.2 Azure VE

### Summary
Please also include information about the reproducibility and the severity/impact of the issue.

### Steps To Reproduce
1. Submit the following declaration:
```json
{
    "class": "AS3",
    "action": "deploy",
    "persist": true,
    "loglevel": "debug",
    "declaration": {
        "class": "ADC",
        "schemaVersion": "3.25.0",
        "id": "*****",
        "TENANT": {
            "class": "Tenant",
            "APP": {

                "backend_nodes": {
                    "class": "Address_Discovery",
                    "addressDiscovery": "azure",
                    "updateInterval": 3600,
                    "tagKey": "tag_name",
                    "tagValue": "tag_value",
                    "addressRealm": "private",
                    "resourceGroup": "*****",
                    "subscriptionId": "*****",
                    "directoryId": "*****",
                    "applicationId": "*****",
                    "apiAccessKey": "*****",
                    "credentialUpdate": false
                },

                "class": "Application",
                "template": "shared"
            }
        }
    }
}
```

2. Observe the following error response:
```json
 {
    "message": "Cannot read property 'forEach' of undefined",
    "host": "localhost",
    "tenant": "WM_WAF",
    "code": 422
}
```

### Expected Behavior
2xx

### Actual Behavior
422 with aforementioned error

**Comment by dstokesf5** [comment]
I have confirmed this bug. It is occurring because the Address_Discovery object is not being used by anything. If you had a pool that referenced /TENANT/APP/backend_nodes, this error would go away. I have added this bug to our internal product backlog as AUTOTOOL-2272.

**Comment by ghalevy** [comment]
@dstokesf5 - Thank you, I can confirm this error goes away once referenced.

**Comment by dstokesf5** [comment]
This appears to have been resolved in the 3.27.0 release.
